src/main/menu.ts

```diff
@@ -455,6 +455,36 @@ export default class MenuBuilder {
                 },
               ],
       },
+      {
```

Comment:
This menu needs to be added on mac as well, currently it's only displayed on linux / windows

Reply:
I couldn't find any documentation on how to add this to mac, and I'm not too sure how I can test it either since I'm on windows. Where should I look in the codebase for this?

Reply:
See how the mac (sometimes called 'darwin') menu is constructed. You wont be able to test it but it should be pretty obvious if it's right or not. It's all in menu.ts

Reply:
Okay think I have added the menu item to mac, let me know if you are able to see it